db/schema.rb

```diff
@@ -56,7 +56,7 @@
     t.string   "primary_language"
     t.string   "call_number"
     t.text     "description"
-    t.text     "physical_description"
+    t.string   "physical_description"
```

Comment:
For my edification, what's the difference between text and string in ActiveRecord?

Reply:
@daryanypl the exact difference depends on the database (MySQL vs. Postgres).  String has a default and max of about 255 characters.  Text has a default and max of about 65,000 characters.  Apparently in Postgres there is no performance advantage: https://stackoverflow.com/questions/3354330/difference-between-string-and-text-in-rails

I removed this code change in a later commit when I noticed it, and you saw that commit which reverts this change.  So this PR as a whole doesn't make a change to the schema for this field.

I don't know the exact reason that my schema was different than what was in source control, but it's not alarming because the one that's in source has `text` rather than `string` so no data will be truncated.